src/lib/utils.ts

```diff
@@ -21,3 +21,26 @@ export function cnTextHw(className: string | undefined) {
     className.includes('text-xs') && 'h-3 w-3',
   ]
 }
+
+/**
+ * Removes headers and HTML tags from a string for use in e.g. meta og:description tags.
+ * @param description - The description to sanitize.
+ * @returns The sanitized description.
+ */
+export function sanitizeDescriptionContent(description: string) {
```

Comment:
Looking good! Is it possible to write some tests for this to make sure it is robust?

Reply:
Removed. Just using taglines instead.